dunelegacy.appdata.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>dunelegacy.desktop</id>
```

Comment:
you put the proper app-id here.

Reply:
It gets renamed to @flathub style.

Reply:
yes but still. this app-id is completely invalid.

Reply:
What is the proper app-id here?

Reply:
according the to manifest **YOU** wrote: `net.sourceforge.DuneLegacy`

Reply:
Note: this doesn't block.

Reply:
I concur though; there's no point in doing it wrong in the downstream repo.

Reply:
My reading of the AppStream metadata is that this _can_ be a reverse domain. Note that this file was created before that was even suggested in the standard.

Reply:
so if it was created before, that mean it is upstream, doesn't it? In that case why is it here?

Reply:
Upstream ignored my patch.